internal/service/cloudbackupsnapshotexportbucket/resource_cloud_backup_snapshot_export_bucket.go

```diff
@@ -248,7 +249,7 @@ func resourceCloudBackupSnapshotExportBucketRefreshFunc(ctx context.Context, cli
 						return nil, "REPEATING", err
 					}
 
-					if s.ChangeStatus == matlas.ChangeStatusPending {
+					if s.GetChangeStatus() == "PENDING" {
```

Comment:
Couldn't find any enum/constant in the new SDK.
Is this the expected way? @lantoli

Reply:
yeah, unfortunately that's the case in this situation.  with the former manual SDK some constants were created manually